rasa/nlu/persistor.py

```diff
@@ -27,15 +30,20 @@ def get_persistor(name: Text) -> Optional["Persistor"]:
             os.environ.get("AZURE_ACCOUNT_NAME"),
             os.environ.get("AZURE_ACCOUNT_KEY"),
         )
-
+    if name:
         try:
             persistor = rasa.utils.common.class_from_module_path(name)
-            return persistor()
         except ImportError:
-            raise ImportError(f"Unknown model persistor {name}. Please make sure to "
-                              f"either use an included model persistor (`aws`, `gcs` "
-                              f"or `azure`) or specify the module path to an external "
-                              f"model persistor.")
+            raise ImportError(
+                f"Unknown model persistor {name}. Please make sure to "
+                "either use an included model persistor (`aws`, `gcs` "
+                "or `azure`) or specify the module path to an external "
+                "model persistor."
+            )
+        else:
+            cfg = os.environ.get("RASA_PERSISTOR")
+            kwargs = json.loads(cfg) if cfg else {}
```

Comment:
in aliyun oss persistor case, i will set the environment variables `RASA_PERSISTOR` to  `'{"access_key_id": "*", "access_key_secret": "*", "sts_role_arn": "*", "region_id": "*", "endpoint": *, "bucket_name": "*", "bucket_path": "*", "local_path": "*"}'`

Reply:
> in aliyun oss persistor case, i will set the environment variables `RASA_PERSISTOR` to `'{"access_key_id": "*", "access_key_secret": "*", "sts_role_arn": "*", "region_id": "*", "endpoint": *, "bucket_name": "*", "bucket_path": "*", "local_path": "*"}'`

@wochinge how do you think about this?

Reply:
or i think the more convenient way to use the external persistor is that getting the init parameter from environment on the persistor lib directly